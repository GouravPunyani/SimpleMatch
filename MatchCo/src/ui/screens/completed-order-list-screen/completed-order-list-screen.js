// @flow

import React, { Component } from 'react'
import { View, Text, StyleSheet } from 'react-native'

export class CompletedOrderListScreen extends Component {
  static propTypes: {
  }

  render () {
    return (
      <View style={styles.wrapper}>
        <Text style={styles.text}>CompletedOrderListScreen</Text>
        <Text style={styles.target}>src/ui/screens/completed-order-list-screen/completed-order-list-screen.js</Text>
      </View>
    )
  }
}

const styles = StyleSheet.create({
  wrapper: {
    flex: 1,
    backgroundColor: 'rebeccapurple',
    justifyContent: 'center',
    alignItems: 'center'
  },
  text: { color: 'white', fontSize: 30 },
  target: { color: 'white', fontSize: 12 }
})
