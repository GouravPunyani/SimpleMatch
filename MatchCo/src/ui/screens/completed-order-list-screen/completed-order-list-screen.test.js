import 'react-native'
import React from 'react'
import renderer from 'react-test-renderer'
import { CompletedOrderListScreen as Screen } from './completed-order-list-screen'

test('renders correctly', () => {
  const tree = renderer.create(<Screen />)
  expect(tree).toMatchSnapshot()
})
