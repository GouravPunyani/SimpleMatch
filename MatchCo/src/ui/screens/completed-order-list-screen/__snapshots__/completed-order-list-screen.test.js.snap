// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "rebeccapurple",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "white",
        "fontSize": 30,
      }
    }
  >
    CompletedOrderListScreen
  </Text>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "white",
        "fontSize": 12,
      }
    }
  >
    src/ui/screens/completed-order-list-screen/completed-order-list-screen.js
  </Text>
</View>
`;
